hazelcast-openshift-centos/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM centos:7
+MAINTAINER Hazelcast, Inc. Integration Team <info@hazelcast.com>
+
+ENV HZ_HOME /opt/hazelcast/
+ENV HZ_CP_MOUNT ${HZ_HOME}/external
+ENV JAVA_HOME /usr/lib/jvm/java
+ENV LANG en_US.utf8
+
+ENV USER_NAME=hazelcast
+ENV USER_UID=10001
+
+ENV HZ_VERSION 3.8.6
+
+ARG HZ_KUBE_VERSION=1.0.0
+ARG HZ_MAVEN_DIR=${HZ_VERSION}
+ARG REPOSITORY_URL=https://repository-hazelcast-l337.forge.cloudbees.com
+
+LABEL name="hazelcast/hazelcast-openshift-centos" \
```

Comment:
Does this image only work for OCP or both OCP and Openshift Origin?

Reply:
It should work on Origin, tested on OCP.

Reply:
Thx!
Can we change the name to `hazelcat-enterprise-openshift-centos`? 
Both folder name and all occurrences in the PR if you agree on the comments in PRD.